posthog/api/session_recording.py

```diff
@@ -162,29 +166,24 @@ def snapshots(self, request: request.Request, **kwargs):
             else None
         )
 
-        # NOTE: This is debug code to try and see what is going on for very specific failures due to unicode errors
+        res = {
+            "result": {
+                "next": next_url,
+                "snapshot_data_by_window_id": session_recording_snapshot_data["snapshot_data_by_window_id"],
+            }
+        }
+
+        # NOTE: We have seen some issues with encoding of emojis, specifically when there is a lone "surrogate pair". See #13272 for more details
```

Comment:
@pauldambra  I did! Reference to issue #13272 which contains context and a link to the DRF bug ;)

Reply:
Not even a link! Way too lazy 🤣